src/profile-logic/sanitize.js

```diff
@@ -267,7 +267,7 @@ function sanitizeThreadPII(
     return null;
   }
 
-  // We need to update the stringTable. It's not possible with UniqueStringArray.
+  // We need to update the stringTable. It's not possible with StringTable.
```

Comment:
Hmm this comment is a bit weird now. But the comment itself was weird in the first place (probably from me, the sanitization code was one of my oldest tasks I think 😄)